pkg/api/serialization_test.go

```diff
@@ -125,7 +124,7 @@ func TestList(t *testing.T) {
 	roundTripSame(t, item)
 }
 
-var nonRoundTrippableTypes = util.NewStringSet("ThirdPartyResource")
```

Comment:
Is this intentional?
Is it required for fixing https://github.com/kubernetes/kubernetes/issues/13083?
If yes, then can you please mention why this needs to be removed in PR description?